project/src/main/world/creature/creature.gd

```diff
@@ -364,7 +364,7 @@ func get_creature_def() -> CreatureDef:
 	var result := CreatureDef.new()
 	result.creature_id = creature_id
 	result.dna = DnaUtils.trim_dna(dna)
-	result.chat_theme_def = chat_theme_def
+	result.chat_theme.from_json_dict(chat_theme.to_json_dict())
```

Comment:
Add a comment describing why we don't pass by reference here

Reply:
Added comment, `# create a copy to prevent our chat theme from being modified accidentally`